api-specificatie/zrc/openapi.yaml

```diff
@@ -1,28 +1,32 @@
 openapi: "3.0.0"
 info:
   title: Zaak registratie API
-  description: Een API om een zaak registratie component te kunnen benaderen
+  description: Een API om een zaakregistratiecomponent te kunnen benaderen
   version: 1.0.0
   contact:
     url:  https://github.com/VNG-Realisatie/gemma-zaken
 paths:
-  /zaken:
+  '/api/v{version}/zaken':
     post:
       summary: Aanmaken nieuwe zaak
-      operationId: aanmakenZaak
-      description: Maakt een nieuwe zaak aan en retourneert de toegekende zaak identificatie wanneer de zaak succesvol is aangemaakt.
+      operationId: zaak_create
```

Comment:
Ik weet niet zo goed hoe operationId in de praktijk van nut is maar lijkt me goed om Nederlands aan te houden, of in ieder geval consistent Nederlands of Engels.

Reply:
Lijkt me prima. Ter referentie, de huidige ZDS 1.1/1.2 naamgevingen zijn:

* ``creeerZaak``
* ``updateZaak``
* ``geefLijstZaken``
* ``geefZaakdetails``
* Er is geen ``verwijderZaak`` oid

Mee eens dat resource + actie het fijnst is. Wellicht heeft VNG er een mening over.

Reply:
`creeerZaak` vind ik bijvoorbeeld erg vreemd - het is geen correct Nederlands (mist trema), en voelt dus ook een beetje als een 'ik-doe-maar-wat' tussenoplossing. My 2 cents :wink:

Reply:
Laten we dit vooruitschuiven. Ik wil niet dat dit de voortgang blokkeert, laat maar zoals het is.